src/Parser/Monk/Mistweaver/Modules/Spells/AzeriteTraits/UpliftedSpirits.js

```diff
@@ -1,18 +1,20 @@
 
 import SPELLS from 'common/SPELLS';
 
+import StatTracker from 'Parser/Core/Modules/StatTracker';
 import Analyzer from 'Parser/Core/Analyzer';
 import Combatants from 'Parser/Core/Modules/Combatants';
 
-import { MISTWEAVER_HEALING_AURA, VIVIFY_SPELLPOWER_COEFFICIENT } from '../../../Constants';
+import { MISTWEAVER_HEALING_AURA, VIVIFY_SPELLPOWER_COEFFICIENT, VIVIFY_REM_SPELLPOWER_COEFFICIENT } from '../../../Constants';
 
 class UpliftedSpirits extends Analyzer {
   static dependencies = {
     combatants: Combatants,
+    statTracker: StatTracker,
   };
 
   healBuff = 0
-
+  critMod = 1
```

Comment:
Does this need to be a member variable?  It looks like you only use it in on_heal and it gets reset every call

Reply:
No it doesn't. I modified to just use the variable within the on_heal event.